frontend/src/pages/Login.vue

```diff
@@ -30,7 +30,7 @@
             </b-col>
           </b-row>
           <b-row>
-            <b-col cols="12" lg="4">
+            <b-col cols="12" lg="6">
```

Comment:
How is this change related to this? Does this addresses our login button problem?

![image](https://user-images.githubusercontent.com/1238238/221043562-0eb02f9f-7362-470d-8af9-34672d268d68.png)

Reply:
this is not related. but i fix the problem with the login submit button with the change.

Reply:
#2738

Reply:
https://github.com/gradido/gradido/pull/2732